xbmc/utils/StringUtils.cpp

```diff
@@ -1810,3 +1810,21 @@ const std::locale& StringUtils::GetOriginalLocale() noexcept
 {
   return g_langInfo.GetOriginalLocale();
 }
+
+void StringUtils::RemoveBetween(std::pair<char, char> bounderies,
```

Comment:
Boundaries

Just a typo in a couple spots

Reply:
whoops